src/main/java/net/sf/jabref/gui/FileExtensions.java

```diff
@@ -12,36 +14,34 @@
 public enum FileExtensions {
```

Comment:
I already thought about it when doing it first in  #1336 and would generally agree, but in this case one entry can have multiple extensions (for example EndNote) and I therefore thought the plural form would reflect it better